applications/nrf_desktop/src/modules/Kconfig.hids

```diff
@@ -31,6 +31,14 @@ config DESKTOP_HIDS_FIRST_REPORT_DELAY
 	  By default, nRF Desktop keyboard uses a delay of 500 ms to prevent
 	  dropping HID reports right after reconnection.
 
+config DESKTOP_HIDS_SUBSCRIBER_PRIORITY
+	int "HID service reports subscriber priority"
+	default 1 # The lowest possible priority in the application
```

Comment:
Maybe we can move the information to help? To make it visible in menuconfig
`By default, the HID service uses the lowest possible priority.`

Similar for USB